packages/zowe-explorer/src/shared/utils.ts

```diff
@@ -473,7 +474,7 @@ export async function promptForEncoding(node: IZoweDatasetTreeNode | IZoweUSSTre
     } else if (node.encoding === null || currentEncoding === "text") {
         currentEncoding = ebcdicItem.label;
     }
-    const encodingHistory = ZoweLocalStorage.getValue<string[]>("zowe.encodingHistory") ?? [];
+    const encodingHistory = ZoweLocalStorage.getValue<string[]>("zowe.encodingHistory") ?? []; //TreeProviders.uss.getEncodingHistory() ?? [];
```

Comment:
Once we move the `"zowe.encodingHistory"` to the enum mentioned in a previous comment, I think we can change it across the board (without causing a circular import)

Also, don't forget to remove any commented out code 😋